networkx/algorithms/bipartite/__init__.py

```diff
@@ -72,7 +72,7 @@
 All bipartite graph generators in NetworkX build bipartite graphs with the 
 "bipartite" node attribute. Thus, you can use the same approach:
 
->>> RB = nx.bipartite_random_graph(5, 7, 0.2)
+>>> RB = nx.algorithms.bipartite.random_graph(5, 7, 0.2)
```

Comment:
This can be `RB = bipartite.random_graph(5, 7, 0.2)` because the bipartite package is already imported in the example.